clusterman/draining/queue.py

```diff
@@ -400,10 +401,14 @@ def terminate_host(host: Host) -> None:
 
 def main(args: argparse.Namespace) -> None:
     setup_config(args)
-    for pool in get_pool_name_list(args.cluster, 'mesos'):
-        load_cluster_pool_config(args.cluster, pool, 'mesos', None)
-    for pool in get_pool_name_list(args.cluster, 'kubernetes'):
-        load_cluster_pool_config(args.cluster, pool, 'kubernetes', None)
+    try:
+        for pool in get_pool_name_list(args.cluster, 'mesos'):
+            load_cluster_pool_config(args.cluster, pool, 'mesos', None)
+        for pool in get_pool_name_list(args.cluster, 'kubernetes'):
+            load_cluster_pool_config(args.cluster, pool, 'kubernetes', None)
+    except ClusterNotFoundError as e:
```

Comment:
Same comment as above; why not just skip the try/except block?

Reply:
I've changed to just log then continue to raise.